### PR TITLE
Коневский -- Лабораторная работа #0

### PR DESCRIPTION
**Pull request by VitKonevskiy**

### CODE REVIEW

**Comment by VitKonevskiy** [comment]
@Vagik
@nik709

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        ComplexNumber z2 = new ComplexNumber(0, 1);
+
+        // Act
+        ComplexNumber z = z1.add(z2);
+
+        // Assert
+        assertEquals(new ComplexNumber(1, 1), z);
+    }
+
+    private final double delta = 0.001;
```
Comment (Vagik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java]: Please move it to top of the class

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+    @Test
+    public void canGetHashCode() {
+        ComplexNumber z1 = new ComplexNumber(7, 7);
+        assertFalse(z1.hashCode() < 0);
+    }
+
+    @Test
+    public void canAddComplexNumbers() {
+        // Arrange
```
Comment (Vagik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java]: Comments are unnecessary

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    private final double delta = 0.001;
+
+    @Test
+    public void canCreateComplexNumberWithInitialValues() {
+        ComplexNumber number = new ComplexNumber(1, 1);
+        assertNotNull(number);
+    }
+
+    @Test
+    public void canSetInitialRealValue() {
```
Comment (Vagik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java]: Can't see any difference between canGetRealValue and canSetInitialRealValue tests

File: code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+    @Test
+    public void areNumbersWithDifferentImaginaryPartNotEqual() {
+        ComplexNumber z1 = new ComplexNumber(3.14, 1.00);
+        ComplexNumber z2 = new ComplexNumber(3.14, 2.73);
+        assertFalse(z1.equals(z2));
+    }
+
+    @Test
+    public void canAddNumbers() {
```
Comment (Vagik) [inline code/korniakov-kirill-lab1/src/test/java/ru/unn/agile/ComplexNumber/Model/KonevskiyVitalyTest.java]: Is duplicate test of canAddComplexNumbers

**Comment by VitKonevskiy** [comment]
@Vagik , I've remove copied tests. Could you review again? Thanks

**Comment by Vagik** [comment]
Looks good for me 👍

**Comment by nik709** [comment]
👍

**Comment by VitKonevskiy** [comment]
@kirill-korniakov

**Pull request merged**